components/SearchBox/components/SeeAll.tsx

```diff
@@ -0,0 +1,30 @@
+import type { Results } from '@orama/orama';
+import type { FC } from 'react';
+
+import type { SearchDoc } from '@/components/SearchBox/components/SearchBox';
+
+import styles from './index.module.css';
+
+type SearchResults = Results<SearchDoc>;
+
+type SeeAllProps = {
+  searchResults: SearchResults;
+  searchTerm: string;
+  selectedFacetName: string;
+};
+
+export const SeeAll: FC<SeeAllProps> = props => {
```

Comment:
Instead of adding this herem the component should only be rendered if there is a search term `searchTerm && <SeAll {...props} />`